### ISSUE TITLE
Code for loading MPSGaze dataset

### ISSUE THREAD

**Issue by Ambitious-idiot** [mf-zhang/GazeOnce#8]
Hello, I want to train a gaze estimator on the MPSGaze dataset. Although there is a short README, I can't find a detailed description of the ranges and meanings of each column, especially for -1 values and out-of-range pitches and yaws. Is it possible to provide a script for data processing or at least a brief description of each columns ( range, default, ...)

**Comment by Ambitious-idiot** [comment]
Especially for the training split

**Comment by mf-zhang** [comment]
We follow the annotation format in [WIDER FACE](https://github.com/biubug6/Pytorch_Retinaface) and in [ETH-XGaze](https://github.com/xucong-zhang/ETH-XGaze) to generate our annotation. If you have further questions, please feel free to leave a message with an example :)

**Comment by Ambitious-idiot** [comment]
Thank you for your explanation, but I'm still confused. For example, there is a data item in the training split with labels like ``1000 25 17 20 1004.482 29.643 1.0 1008.232 30.714 1.0 1000.732 34.598 1.0 1002.205 38.348 1.0 1003.946 38.884 1.0 0.27``. Compared to the example provided in the data description, does it mean that the gaze pitch and yaw are 1.0 and 1003.946 respectively?

**Comment by mf-zhang** [comment]
Thank you for the question!

In this example, `1000 25 17 20 ` is face bbox, `1004.482 29.643 1.0 1008.232 30.714 1.0 1000.732 34.598 1.0 1002.205 38.348 1.0 1003.946 38.884 1.0` is five landmarks. Because this face is too small, we don't do face swapping, and there is no gaze label. 

There are similar cases in [WIDER FACE](https://github.com/biubug6/Pytorch_Retinaface), where some of the faces are too small to have landmark labels. In their training code, such faces are not counted in the landmark loss.

Our training process follows [WIDER FACE](https://github.com/biubug6/Pytorch_Retinaface) to handle the label missing problem. I will add this explanation in the README. Thanks for asking!

**Comment by Ambitious-idiot** [comment]
Thank you for your detailed explanation! I understand the solution now.
